modules/nf-core/samtools/sormadup/main.nf

```diff
@@ -0,0 +1,67 @@
+process SAMTOOLS_SORMADUP {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::samtools=1.16.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.16.1--h6899075_1' :
+        'quay.io/biocontainers/samtools:1.16.1--h6899075_1' }"
+
+    input:
+    tuple val(meta), path(input)
+    tuple val(meta2), path(fasta)
+
+    output:
+    tuple val(meta), path("*.bam")      , emit: bam
+    tuple val(meta), path("*.metrics")  , emit: metrics
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
```

Comment:
Don't we need another args for at least the initial `samtools cat` ? (perhaps for all the other `samtools` commands too)

Reply:
I'm not really sure, I'd like to keep this as simple as possible to avoid any confusion. I do agree it would be good for me modifiability though.

Reply:
My main target here it to get a samtools equivalent for `BIOBAMBAM_BAMSORMADUP`, that's why I went with the module route

Reply:
Added the extra `ext.args`